src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -79,6 +79,13 @@
     <!-- Only pass when enabled to reduce command line noise. -->
     <BuildArgs Condition="'$(DotNetBuildTests)' == 'true'">$(BuildArgs) /p:DotNetBuildTests=true</BuildArgs>
     <BuildArgs Condition="'$(NuGetConfigFile)' != ''">$(BuildArgs) /p:RestoreConfigFile=$(NuGetConfigFile)</BuildArgs>
+
+    <BuildArgs Condition="'$(UseSystemBrotli)' != ''">$(BuildArgs) /p:UseSystemBrotli=$(UseSystemBrotli)</BuildArgs>
+    <BuildArgs Condition="'$(UseSystemLibunwind)' != ''">$(BuildArgs) /p:UseSystemLibunwind=$(UseSystemLibunwind)</BuildArgs>
+    <BuildArgs Condition="'$(UseSystemLlvmLibunwind)' != ''">$(BuildArgs) /p:UseSystemLlvmLibunwind=$(UseSystemLlvmLibunwind)</BuildArgs>
+    <BuildArgs Condition="'$(UseSystemRapidjson)' != ''">$(BuildArgs) /p:UseSystemRapidjson=$(UseSystemRapidjson)</BuildArgs>
+    <BuildArgs Condition="'$(UseSystemZlib)' != ''">$(BuildArgs) /p:UseSystemZlib=$(UseSystemZlib)</BuildArgs>
```

Comment:
Is it better to add specific named options like this? Or better to have generic mechanism to pass any property to the inner build?

Reply:
Could we use a property named something like `UseSystemLibs` and have the names specified as a `+`-deliminated list? Like `/p:UseSystemLibs=brotli+libunwind+rapidjson+zlib`? And then implement that property as the one that's passed through?